tests/integration/handling-api-messages.js

```diff
@@ -0,0 +1,20 @@
+describe("handling api messages", function () {
+
+  it("handles and re-hydrates commands", function () {
```

Comment:
That's the question, in this case there is no validation or anything going on. We also have no helper code to deal with validations etc. at all. So i understand your idea but here we are just testing the technical side of things (e.g: does it work, or not). Also `Space.messaging` has nothing todo with CQRS / ES and i think that's important.

If we add code that makes validating stuff easier (like before-action hooks in rails) than your tests would make sense. But honestly, your test descriptions are even more confusing, because people would *think*  that there is some validation going on (which is not).

The only thing *that is going on* is EJSON (de)-serialization and a simple `check`.